client/src/templates/Challenges/classic/mobile-layout.tsx

```diff
@@ -45,6 +45,65 @@ class MobileLayout extends Component<MobileLayoutProps, MobileLayoutState> {
     });
   };
 
+  avoidWindowScrolling = (event: Event) => {
+    event.preventDefault();
+  };
+
+  setToolPanelPosition = () => {
+    const toolPanelGroup = (
+      document.getElementsByClassName(
+        'tool-panel-group-mobile'
+      ) as HTMLCollectionOf<HTMLElement>
+    )[0];
+
+    if (navigator.userAgent.match(/iPhone|Android.+Mobile/)) {
+      if (window.innerHeight > (visualViewport?.height as number)) {
```

Comment:
Is overwriting the `visualViewport` type intended? and what is the benefits over checking if the value exist something like this 

```suggestion
      if (visualViewport.height && window.innerHeight > visualViewport.height) {
```

Reply:
Sorry, the [visualViewport.height](https://developer.mozilla.org/en-US/docs/Web/API/VisualViewport/height) is always a double, so type overwriting is unnecessary . (I overwrote the type because of just avoiding TypeScript error in code editor)